WORKSPACE

```diff
@@ -117,6 +117,8 @@ distdir_tar(
         "8ccf4f1c351928b55d5dddf3672e3667f6978d60.tar.gz",
         "0.16.2.zip",
         "android_tools_pkg-0.5.tar.gz",
+        # bazelbuild/platforms
+        "d91995f1a9497602a02d11db41a7ffa5551147be.zip"
```

Comment:
Does the filename need to be opaque?

Reply:
Nope, but I didn't care here, the filename will make sense once we have an actual release archive of bazelbuild/platforms.

Reply:
SGTM.

Reply:
Is it worth doing a quick 0.1.0 release of platforms to clean the name up now.
If not, it makes it harder to find and update line 174 (  "d91995f1a9497602a02d11db41a7ffa5551147be.zip": ) than it would be if it were "platforms-0.1.0.tar.gz"

Reply:
and you won't need the strip prefix at line 514

Reply:
I decided not to block submission of this PR on setting up release pipeline for platforms. We definitely have to set up releases for platforms repository. I filed https://github.com/bazelbuild/continuous-integration/issues/715.